docs/src/content/docs/challenges/angular/55-back-button-navigation.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 🟠 Back-Button-Navigation
+description: Challenge 55 is about overriding browser back button navigation
+author: Ioannis-Tsironis
+contributors:
+  - tomalaforge
```

Comment:
can remove this

Reply:
What to remove here?

Reply:
You can remove my name, I haven't contributed to your challenge yet 😅